.github/workflows/ttnn-post-commit.yaml

```diff
@@ -60,6 +60,8 @@ jobs:
             fast_runtime_mode_off: true
           - name: ttnn examples and cpp tests
             cmd: ./build/test/ttnn/unit_tests_ttnn && ./tests/scripts/run_ttnn_examples.sh
+          - name: ttnn ccl cpp unit tests
+            cmd: ./build/test/ttnn/unit_tests_ttnn_ccl
```

Comment:
please add this to one of the scripts called by `run_test.sh` so that devs who want to run locally pick it up also